Tools/autotest/common.py

```diff
@@ -1061,6 +1138,12 @@ def test_arm_feature(self):
         if not self.disarm_vehicle():
             self.progress("Failed to DISARM")
             raise NotAchievedException()
+        if not self.arm_vehicle_with_mavproxy():
```

Comment:
Should we have a pattern of tests that we just do with mavproxy as `mavproxy_arm_vehicle`?